pkg/security/ebpf/byteorder.go

```diff
@@ -0,0 +1,31 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-2020 Datadog, Inc.
+
+package ebpf
+
+import (
+	"encoding/binary"
+	"unsafe"
+)
+
+// GetHostByteOrder guesses the hosts byte order
+func GetHostByteOrder() binary.ByteOrder {
```

Comment:
Should this be in `pkg/util` instead?

Reply:
This is related to eBPF so it's probably ok here